airlock/management/commands/create_release_request.py

```diff
@@ -65,6 +66,19 @@ def add_arguments(self, parser):
     def handle(self, username, workspace_name, **options):
         user = User.objects.get(user_id=username)
         request = bll.get_or_create_current_request(workspace_name, user)
+        if request.is_under_review():
+            self.stdout.write(
+                f"A release request for workspace '{workspace_name}' is already under review for user '{username}'"
+            )
+            # Management commands expect to write their output to std out, so return it as a (json)string
+            return json.dumps(
```

Comment:
What happens with this return value if we run this command manually from the cli?

Does it get printed? Or just swallowed? Does it exit with 0 or 1?